packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -233,7 +233,36 @@ If there is a response timeout after receiving the Okta request, the Okta proces
 }
 ```
 
-## Enabling a Registration Inline Hook for Self-Service Registration
+## Enabling a Registration Inline Hook for Profile Enrollment
+
+<ApiLifecycle access="ie" /><br>
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+To activate the Inline Hook, you first need to register your external service endpoint with Okta; see [Inline Hook Setup](/docs/concepts/inline-hooks/#inline-hooks_setup).
+
+You then need to associate the registered Inline Hook with your Profile Enrollment policy. (For information on configuring a Profile Enrollment policy, see [Enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm).)
+
+1. Go to **Security > Profile Enrollment**
+
+1. Click the pencil icon to edit the policy and associate it with your Registration Inline Hook.
```

Comment:
Are these numbered items supposed to increment?

Reply:
they become incremental when the markdown is rendered